praw/objects.py

```diff
@@ -698,8 +698,28 @@ def from_id(reddit_session, message_id, *args, **kwargs):
             if child.id == message_id:
                 return child
 
+    @restrict_access(scope='privatemessages')
+    def collapse(self):
+        """Collapse a private message or modmail
+
+        """
+
+        data = {'id': self.name}
+        url = 'https://api.reddit.com/api/collapse_message/'
```

Comment:
You should add these urls to [the api paths](https://github.com/praw-dev/praw/blob/master/praw/__init__.py#L83) config in the form seen there. Then you can replace these lines with `url = self.reddit_session.config['collapse_message']`. Just make sure to keep them alphabetized.